ocp_install_env.sh

```diff
@@ -130,6 +130,22 @@ EOF
   fi
 }
 
+function apiVIP_ingressVIP_dualstack() {
```

Comment:
Nit: this isn't specific to dual stack. This function will be called for all deployment types.